src/types/lib.ts

```diff
@@ -24,6 +26,7 @@ export interface IProxyCache {
   disconnect: () => Promise<boolean>;
   healthCheck: () => Promise<boolean>;
   isConnected: boolean;
+  client: ProxyClientType;
```

Comment:
Why do we expose internal details of _ProxyCache_ impl.?  Any clients of _proxyCache lib_ should have access only to defined methods, not internals